### PR TITLE
[HOPSWORKS-3100] Add feature view migration

### PR DESCRIPTION
**Pull request by kennethmhc**

### CODE REVIEW

File: templates/default/expat-site.xml.erb [hunk 1]
```diff
@@ -147,4 +147,7 @@
         io.hops.hopsworks.expat.migrations.featurestore.storageconnectors.StorageConnectorMigration
         io.hops.hopsworks.expat.migrations.featurestore.trainingdataset.CreateExternalTrainingDatasetInode
     </version-26>
+    <version-30>
```
Comment (maismail) [inline templates/default/expat-site.xml.erb:150]: It should be still under <version-26>, we should rename the whole tag once 3.0 is planned for release

**Review by maismail — CHANGES_REQUESTED** [review]
@kennethmhc you would need to add the same to https://github.com/logicalclocks/hopsworks-chef/blob/master/templates/default/expat-site.xml.erb

File: templates/default/expat-site.xml.erb (hunk 1 above)
Comment (kennethmhc) [reply templates/default/expat-site.xml.erb:150]: @maismail do you mean it should be put under `version-26`?

Comment (maismail) [reply templates/default/expat-site.xml.erb:150]: yes

**Review by maismail — APPROVED** [review]

**Pull request merged**